tests/bunq_test.py

```diff
@@ -15,33 +25,130 @@ class BunqSdkTestCase(unittest.TestCase):
     # Device description used for python tests
     _DEVICE_DESCRIPTION = 'Python test device'
 
+    _PATH_ATTACHMENT = '/assets/'
+    _READ_BYTES = "rb"
+    _ATTACHMENT_PATH_IN = 'bunq_App_Icon_Square@4x.png'
+    _CONTENT_TYPE = 'image/png'
+    _ATTACHMENT_DESCRIPTION = 'SDK python test'
+    _FIRST_INDEX = 0
+
+    _second_monetary_account = None
+    _cash_register = None
+
     @classmethod
-    def _get_api_context(cls):
-        """
-        Calls IsSessionActive to check if the session token is still active
-        and returns the context.ApiContext.
+    def setUpClass(cls):
+        BunqContext.load_api_context(cls._get_api_context())
+
+    def setUp(self):
+        self.__set_second_monetary_account()
+        self.__request_spending_money()
+        time.sleep(0.5)
+        BunqContext.user_context().refresh_user_context()
+
+    def __set_second_monetary_account(self):
+        response = endpoint.MonetaryAccountBank.create(
+            'EUR',
+            'test account python'
+        )
+
+        self._second_monetary_account = endpoint.MonetaryAccountBank.get(
+            response.value
+        ).value
 
-        Catches ApiException if the session is inactive.
-        Catches BunqException if the conf file does not exist.
+    def __request_spending_money(self):
+        endpoint.RequestInquiry.create(
+            object_.Amount('500', 'EUR'),
+            object_.Pointer('EMAIL', 'sugardaddy@bunq.com'),
+            'sdk  python test, thanks daddy <3 - OG',
```

Comment:
double space between `sdk` and `python`, also remove `- OG` to keep in line with PHP.